src/replit_river/client.py

```diff
@@ -27,6 +31,21 @@
 tracer = trace.get_tracer(__name__)
 
 
+@dataclass(frozen=True)
+class RiverUnknownValue(BaseModel):
+    tag: Literal["RiverUnknownValue"]
+    value: Any
+
+
+def raise_unknown(
```

Comment:
I was sort of confused by this name, since the implementation doesn't actually raise anything.

Reply:
I can change it, I was going for "raise an exceptional value" but I can see how it's confusing

Reply:
Going with `translate_unknown_value`